### DISCUSSION TITLE
Additional Site Alert (email report)

### DISCUSSION THREAD

**Discussion by Jereami** [plausible/analytics#1320] [Anything Else]
I ran into an issue where a Cloudflare worker was responding with a 404 on my proxied install of Plausible. Unfortunately I didn't notice it for a about a day. 

I know we have the ability to send email on traffic spikes; so is it possible to easily add the inverse where alerts are sent on traffic declines (X visitors over Y time)?

**Comment by metmarkosaric** [comment]
thanks @Jereami! that would be nice feature to have! there are couple of similar threads already in #962 and #474

**Reply by Jereami** [reply]
My apologies, I missed those.

**Reply by metmarkosaric** [reply]
np @Jereami!
